### PR TITLE
Add dymension and saga configs

### LINKED ISSUE #161: Add Dymension chain support

### PR DESCRIPTION
**Pull request by 0xh3rman**
Fixes #161

### CODE REVIEW

File: crates/primitives/src/chain.rs [hunk 1]
```diff
@@ -118,6 +124,46 @@ impl Chain {
         }
     }
 
+    pub fn hrp(&self) -> &str {
```
Comment (gemcoder21) [inline crates/primitives/src/chain.rs:127]: There is `CosmosChain` type, maybe have `hrp` under it?

File: crates/primitives/src/chain.rs [hunk 2]
```diff
@@ -138,13 +184,19 @@ impl Chain {
             | Self::ZkSync
             | Self::Linea
             | Self::Mantle
-            | Self::Celo => 60,
+            | Self::Celo
+            | Self::Dymension => 60,
```
Comment (gemcoder21) [inline crates/primitives/src/chain.rs:188]: that's unusual for cosmos chain to pick `60`

File: crates/primitives/src/explorer.rs [hunk 3]
```diff
@@ -209,6 +209,18 @@ impl<'a> Explorer<'a> {
...
+            Chain::Dymension => Explorer {
+                host: "https://www.mintscan.io/dymension",
+                transaction_url: "/tx/",
+                account_url: "/address/",
+                token_url: Some("/assets/"),
+            },
+            Chain::Saga => Explorer {
+                host: "https://www.mintscan.io/saga",
+                transaction_url: "/tx/",
+                account_url: "/address/",
```
Comment (gemcoder21) [inline crates/primitives/src/explorer.rs:221]: does token url works? I think it should be `None`

**Review by gemcoder21 — APPROVED** [review]

File: crates/primitives/src/chain.rs (hunk 1 above)
Comment (0xh3rman) [reply crates/primitives/src/chain.rs:127]: good point

File: crates/primitives/src/chain.rs (hunk 2 above)
Comment (0xh3rman) [reply crates/primitives/src/chain.rs:188]: dimension has ETH address too, that's why it uses 60, similar to Evmos and injective

File: crates/primitives/src/explorer.rs (hunk 3 above)
Comment (0xh3rman) [reply crates/primitives/src/explorer.rs:221]: nice catch, saga doesn't have tokens yet, we can use `None` for now

**Pull request merged**